### PR TITLE
[TEST] add seqan2 chunked output to benchmark

### PR DESCRIPTION
**Pull request by h-2**
SeqAn2 could do chunked output already, but SeqAn3's implementation is ~5% faster.

### CODE REVIEW

File: test/performance/io/lowlevel_stream_output_benchmark.cpp [hunk 1]
```diff
@@ -40,6 +41,9 @@ void write_all(benchmark::State & state)
     // sequence to write:
     std::vector<char> cont_rando = seqan3::test::generate_sequence<char>(10'000, 0, 0);
 
+#ifdef SEQAN3_HAS_SEQAN2
+    auto cont_rando2 = seqan3::test::generate_sequence_seqan2<char>(10'000, 0, 0);
```
Comment (smehringer) [inline test/performance/io/lowlevel_stream_output_benchmark.cpp]: The comparison should be fair even though the sequence `cont_rando` and `cont_rando2` will differ right?

Comment (h-2) [reply test/performance/io/lowlevel_stream_output_benchmark.cpp]: Yes! There is no difference. It's only important that it's not all zeros.

Comment (rrahn) [reply test/performance/io/lowlevel_stream_output_benchmark.cpp]: Maybe reconsider better names for `cont_rando`. This sounds like a bad name for a 🐶

Comment (h-2) [reply test/performance/io/lowlevel_stream_output_benchmark.cpp]: It *is* the name of my :dog2: . Don't mess with it.

Comment (rrahn) [reply test/performance/io/lowlevel_stream_output_benchmark.cpp]: that pure little thing. 😄

Comment (rrahn) [inline test/performance/io/lowlevel_stream_output_benchmark.cpp:45]:
```suggestion
    auto seqan2_sequence = seqan3::test::generate_sequence_seqan2<char>(10'000, 0, 0);
```

File: test/performance/io/lowlevel_stream_output_benchmark.cpp [hunk 2]
```diff
@@ -40,6 +41,9 @@ void write_all(benchmark::State & state)
     // sequence to write:
     std::vector<char> cont_rando = seqan3::test::generate_sequence<char>(10'000, 0, 0);
```
Comment (rrahn) [inline test/performance/io/lowlevel_stream_output_benchmark.cpp:42]:
```suggestion
    std::vector<char> sequence = seqan3::test::generate_sequence<char>(10'000, 0, 0);
```

File: test/performance/io/lowlevel_stream_output_benchmark.cpp [hunk 3]
```diff
@@ -71,6 +75,12 @@ void write_all(benchmark::State & state)
         {
             it.write_range(cont_rando);
```
Comment (rrahn) [inline test/performance/io/lowlevel_stream_output_benchmark.cpp:76]:
```suggestion
            it.write_range(sequence);
```

File: test/performance/io/lowlevel_stream_output_benchmark.cpp [hunk 4]
```diff
@@ -71,6 +75,12 @@ void write_all(benchmark::State & state)
...
         }
+        #ifdef SEQAN3_HAS_SEQAN2
+        else if constexpr (id == tag::seqan2_stream_it_write_range)
+        {
+            seqan::write(it, cont_rando2);
```
Comment (rrahn) [inline test/performance/io/lowlevel_stream_output_benchmark.cpp:81]:
```suggestion
            seqan::write(it, seqan2_sequence);
```

**Pull request merged**